homeassistant/components/blebox/switch.py

```diff
@@ -1,16 +1,19 @@
 """BleBox switch implementation."""
 from datetime import timedelta
 
-from homeassistant.components.switch import SwitchEntity
+from homeassistant.components.switch import SwitchDeviceClass, SwitchEntity
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
 from . import BleBoxEntity, create_blebox_entities
-from .const import BLEBOX_TO_HASS_DEVICE_CLASSES
 
 SCAN_INTERVAL = timedelta(seconds=5)
 
+BLEBOX_TO_SWITCH_DEVICE_CLASSES = {
+    "relay": SwitchDeviceClass.SWITCH,
+}
+
```

Comment:
If these new dict have only one value, are they actually needed?
Are there plans to add more items to them? or do they just bloat the code base?

Reply:
Yes, I've just followed data structure. In `switch` and `button` platforms, I'll remove. In `sensor` platform there are plan for new sensor and AirQuality will be migrated there. `cover` platform finds this structure useful.